### PR TITLE
Move two tutorials from `aiida-core` to SINTEF appendices

### PR DESCRIPTION
**Pull request by sphuber**
Two pages are added to the appendix of SINTEF that are in depth
tutorials about the `StructureData` and `UpfData` class and how to work
with them. These are moved from the `aiida-core` documentation as they
are too plugin specific for `aiida-core`.

### CODE REVIEW

**Comment by ltalirz** [comment]
@yakutovicha Since I already saw the "tutorial_india"  branch, please note that there are a number of updates in the Sintef tutorial with respect to the May tutorial that make sense to keep migrating forward.

File: docs/pages/2019_SINTEF/sections/verdi_shell.rst [hunk 1]
```diff
@@ -294,6 +294,7 @@ repository such as the Crystallography Open Database (COD):
         print("Unit cell volume: ", structure.get_cell_volume())
 
 In that case two duplicate structures are found for 'Si'.
+A more in depth tutorial can be found in :ref:`this appendix<appendix_structure_data>`.
```
Comment (ltalirz) [inline docs/pages/2019_SINTEF/sections/verdi_shell.rst]:
```suggestion
A more in depth tutorial can be found in :ref:`this appendix<2019_sintef_appendix_structure_data>`.
```

**Comment by sphuber** [comment]
There is one problem with this PR. Since the pages I am adding contain references to `aiida-core` resources, such as the `StructureData` class, the build fails because it warns that it cannot find these. I think it is possible to reference across packages using intersphinx. Have you ever used this @ltalirz ?

File: docs/pages/2019_SINTEF/sections/verdi_shell.rst [hunk 2]
```diff
@@ -410,6 +411,8 @@ the database with
 
     verdi data upf listfamilies
 
+A more in depth tutorial about working with `UpfData` nodes and pseudo potenial families can be found in :ref:`this appendix<appendix_upf_data>`.
```
Comment (ltalirz) [inline docs/pages/2019_SINTEF/sections/verdi_shell.rst]:
```suggestion
A more in depth tutorial about working with `UpfData` nodes and pseudo potenial families can be found in :ref:`this appendix<2019_sintef_appendix_upf_data>`.
```

File: docs/pages/2019_SINTEF/sections/appendix_structure_data.rst [hunk 3]
```diff
@@ -0,0 +1,280 @@
+.. _appendix_structure_data:
```
Comment (ltalirz) [inline docs/pages/2019_SINTEF/sections/appendix_structure_data.rst]:
```suggestion
.. _2019_sintef_appendix_structure_data:
```

File: docs/pages/2019_SINTEF/sections/appendix_upf_data.rst [hunk 4]
```diff
@@ -0,0 +1,120 @@
+.. _appendix_upf_data:
```
Comment (ltalirz) [inline docs/pages/2019_SINTEF/sections/appendix_upf_data.rst]:
```suggestion
.. _2019_sintef_appendix_upf_data:
```

**Review by ltalirz — CHANGES_REQUESTED** [review]
thanks @sphuber !
just a few modifications of the references (didn't check whether it compiles).

otherwise good to go

**Comment by ltalirz** [comment]
> I think it is possible to reference across packages using intersphinx. Have you ever used this @ltalirz ?

Indeed, I think one can do this.

I've never tried, but I think you can try adding something like this to the `conf.py`:
```python
extensions = [
    'sphinx.ext.intersphinx',
]

intersphinx_mapping = {
    'aiida':  ('https://aiida-core.readthedocs.io/en/latest/', None),
}
```

**Comment by sphuber** [comment]
Got intersphinx working and adapted the link references you requested. Should be good to go @ltalirz

**Pull request merged**